assets/prefabs/structures/Saw.prefab

```diff
@@ -0,0 +1,809 @@
+            { "blockType": "core:stone", "region": { "min": [18, 4, 2], "size": [4, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [25, -9, 2], "size": [1, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [26, -8, 2], "size": [5, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [27, -9, 2], "size": [4, 1, 2]}},
+            { "blockType": "core:stone", "region": { "min": [8, -15, 3], "size": [15, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [9, -14, 3], "size": [6, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [9, -13, 3], "size": [7, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [11, -11, 3], "size": [1, 2, 1]}},
+            { "blockType": "core:stone", "region": { "min": [11, -9, 3], "size": [2, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [12, -8, 3], "size": [1, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [13, -11, 3], "size": [1, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [13, -7, 3], "size": [18, 3, 1]}},
+            { "blockType": "core:stone", "region": { "min": [14, -8, 3], "size": [17, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [14, -4, 3], "size": [17, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [14, -3, 3], "size": [14, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [15, -9, 3], "size": [10, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [15, -1, 3], "size": [10, 1, 2]}},
+            { "blockType": "core:stone", "region": { "min": [16, -11, 3], "size": [15, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [16, 0, 3], "size": [8, 2, 1]}},
+            { "blockType": "core:stone", "region": { "min": [17, -10, 3], "size": [14, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [17, 2, 3], "size": [7, 2, 1]}},
+            { "blockType": "core:stone", "region": { "min": [18, 4, 3], "size": [6, 1, 2]}},
+            { "blockType": "core:stone", "region": { "min": [24, -15, 3], "size": [7, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [8, -15, 4], "size": [14, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [8, -14, 4], "size": [23, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [9, -13, 4], "size": [22, 1, 2]}},
+            { "blockType": "core:stone", "region": { "min": [10, -12, 4], "size": [21, 2, 1]}},
+            { "blockType": "core:stone", "region": { "min": [11, -10, 4], "size": [3, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [11, -9, 4], "size": [14, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [12, -8, 4], "size": [13, 1, 1]}},
+            { "blockType": "core:stone", "region": { "min": [12, -7, 4], "size": [19, 1, 1]}},
+   
```

Comment:
Before "AddItemsToChest" you need to close the other array and component first properly.